src/DGtal/dec/PolygonalCalculus.h

```diff
@@ -465,54 +477,57 @@ class PolygonalCalculus
     if (checkCache(L_,f))
       return myGlobalCache[L_][f];
  
-    auto Df = D(f);
-    DenseMatrix op = Df.transpose() * M(f,lambda) * Df;
+    DenseMatrix Df = D(f);
+    // Laplacian is a negative operator.
+    DenseMatrix op = -1.0 * Df.transpose() * M(f,lambda) * Df;
```

Comment:
Same here, the convention should be described in the method comment (I cannot suggest edits from GitHub)